plugins/chain_plugin/include/eosio/chain_plugin/chain_plugin.hpp

```diff
@@ -657,9 +673,12 @@ class read_write {
      using index_type = chain::index256_index;
      static auto function() {
         return [](const input_type v) {
+            // The input is in little endian of uint256_t, i.e. fb54b91bfed2fe7fe39a92d999d002c550f0fa8360ec998f4bb65b00c86282f5
+            // fixed_bytes will convert the input to array of 2 uint128_t in little endian, i.e. 50f0fa8360ec998f4bb65b00c86282f5 fb54b91bfed2fe7fe39a92d999d002c5
+            // which is the format used by secondary index
             chain::key256_t k;
-            k[0] = ((uint128_t *)&v)[0]; //0-127
-            k[1] = ((uint128_t *)&v)[1]; //127-256
+            k[0] = ((uint128_t *)&v)[1]; //128-255
```

Comment:
`boost::multiprecision::uint256_t` is a class type, so this is a strict-aliasing violation.